src/app/shared/components/filters/filters.component.ts

```diff
@@ -23,6 +24,7 @@ export class FiltersComponent {
   private _filters: Filter[] = [];
   private _filterChange: EventEmitter<any> = new EventEmitter<any>();
   private _isCollapsed: boolean;
+  private _appliedFilters: Filter[];
```

Comment:
I suggest to initialize by default values of types like object, array, Set, Map, to avoid possible call errors in templates or in async methods of the component.